connector/mqtt/vernemq/v2k-bridge/README.md

```diff
@@ -9,59 +14,82 @@ The V2K-bridge service implements a bridge between VerneMQ broker and Kafka brok
+V2K_MQTT_TLS_KEY_FILE                  | MQTT client key file                                      | ${V2K_APP_BASEDIR}/app/verne/${V2K_APP_BASEDIR}.key | string                         |
+
+### **Microservice SDK configurations**
+
+You can configure the Kafka Producer with variables from the Microservice SDK and librdkafka. For
+more details on these configurations, please read the
+[librdkafka official configuration guide](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
 and [Microservice SDK documentation](https://www.npmjs.com/package/@dojot/microservice-sdk).
 
 SDK configurations:
 
-Key                                | Default Value   |
----------------------------------- | --------------- |
-V2K_PRODUCER_CONNECT_TIMEOUT_MS    | 5000            |
-V2K_PRODUCER_DISCONNECT_TIMEOUT_MS | 10000           |
-V2K_PRODUCER_FLUSH_TIMEOUT_MS      | 2000            |
-V2K_PRODUCER_POOL_INTERVAL_MS      | 100             |
-
-librdkafka configurations:
-
-Key                                             | Default Value     |
------------------------------------------------ | ----------------- |
-V2K_KAFKA_ACKS                                  | -1                |
-V2K_KAFKA_BATCH_NUM_MESSAGES                    | 10000             |
-V2K_KAFKA_CLIENT_ID                             | ${HOSTNAME}       |
-V2K_KAFKA_COMPRESSION_CODEC                     | none              |
-V2K_KAFKA_DR_CB                                 | true              |
-V2K_KAFKA_ENABLE_IDEMPOTENCE                    | false             |
-V2K_KAFKA_MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION | 1000000           |
-V2K_KAFKA_METADATA_BROKER_LIST                  | kafka-server:9092 |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_KBYTES            | 1048576           |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_MS                | 0.5               |
-V2K_KAFKA_RETRIES                               | 2                 |
-V2K_KAFKA_RETRY_BACKOFF_MS                      | 100               |
-V2K_KAFKA_SOCKET_KEEPALIVE_ENABLE               | false             |
\ No newline at end of file
+#### **Logger**
+
+Key                                 | Default Value |
+----------------------------------- | ------------- |
+V2K_LOGGER_TRANSPORTS_CONSOLE_LEVEL | info          |
+V2K_LOGGER_VERBOSE                  | false         |
+
+#### **Producer**
+
+Key                                    | Default Value |
+-------------------------------------- | ------------- |
+V2K_SDK_PRODUCER_CONNECT_TIMEOUT_MS    | 5000          |
+V2K_SDK_PRODUCER_DISCONNECT_TIMEOUT_MS | 10000         |
+V2K_SDK_PRODUCER_FLUSH_TIMEOUT_MS      | 2000          |
+V2K_SDK_PRODUCER_POOL_INTERVAL_MS      | 100           |
+
+#### **librdkafka**
+
+Key                                             | Default Value      |
+----------------------------------------------- | ------------------ |
+V2K_KAFKA_ACKS                                  | -1                 |
```

Comment:
Not in this case, but I have a doubt of how this will work when a service has both consumer and producer. We can have some kafka parameters for the producer and others for the producers.

Reply:
You mean the variable names or the table itself?

Reply:
We can discuss this when we develop the ConfigManager.